provisioning_client/CMakeLists.txt

```diff
@@ -58,9 +58,9 @@ elseif (${use_prov_client})
         include_directories(${TPM_C_INC_FOLDER})
 
         # Include the riot directories
-        include_directories(./deps/RIoT/Emulator/DICE)
-        include_directories(./deps/RIoT/Emulator/RIoT)
-        include_directories(./deps/RIoT/Emulator/RIoT/RIoTCrypt/include)
+        include_directories(./deps/RIoT/Simulation/DICE)
```

Comment:
No change necessary here but a "consider for the future" would be to swap over to the Reference tree and away from the Simulation tree. The sources are the same but the organization is a little different and the sets of sources serve slightly different purposes.

Reply:
Noted